OcCo_TF/README.md

```diff
@@ -1,4 +1,42 @@
-## OcCo in TensorFlow
+## Occlusion Completion: TensorFlow Implementation
 
+This is the `TensorFlow` implementation for Occlusion Completion (OcCo). It is the primary framework for Occlusion Completion; the `PyTorch` implementation has not yet been tested.
 
+### Setup
 
+Create a `conda` environemnt with Python 3.7 named `occo-tf` and run the following installation commands:
```

Comment:
"environment"

Reply:
Also, can we just add the `conda create` instruction you used (with the exact Python version), so long as we are adding other `conda` instructions below.